Standards-Documentation.md

```diff
@@ -339,21 +339,38 @@ Paragraph 1 text
 Paragraph 2 text
 ```
 
+## Link new files to README.md
+
+Before testing your changes make sure you add the necessary links to README.md
+for any new files
+
+Running this script will help you check for any missing links.
+
+```console
+bin/check_missing_link.sh
+```
+
 ## Testing changes
 
 To locally test the changes made to any Markdown document, you will need to
-have installed ruby then run:
+have ruby installed and then run:
 
 ```console
 gem install mdl
 ```
 
+And then run or simply add the line to your `.bash_profile` file
+
+```console
+scl enable rh-ruby22 bash
+```
+
```

Comment:
Why is this a requirement?  I don't recall doing this and things worked for me?

Reply:
Let me go through yum history and check how did I get it installed but `which` can't find it.

I thought it was because I was logged as root but it's the same with dummy user I created. Thing is, didn't work right out of the box.

```console
which: no mdl in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/opt/rh/rh-ruby22/root/usr/local/bin/:/root/bin)
ERROR: mdl (markdownlint) not installed

Check out https://github.com/markdownlint/ for installation instructions
```

```shell
[root@cacti12 documentation]# ls -al /opt/rh/rh-ruby22/root/usr/local/bin/mdl
-rwxr-xr-x. 1 root root 499 Sep  3 07:51 /opt/rh/rh-ruby22/root/usr/local/bin/mdl
[root@cacti12 documentation]# ls -al /opt/rh/rh-ruby22/root/usr/bin/mdl
lrwxrwxrwx. 1 root root 40 Sep  3 19:11 /opt/rh/rh-ruby22/root/usr/bin/mdl -> /opt/rh/rh-ruby22/root/usr/local/bin/mdl
[root@cacti12 documentation]#
```

Reply:
I'll get rid of it and if someone hits that can google it and find its own way out. For the use I give it, I won't bother

Reply:
So, `gem install mdl` is the part that actually installs MDL.  This should enable it by default.  However, your `scl` command is ensuring that software collection of ruby is available to bash... MDL requires Ruby but that should already be available on the system so I'm not sure - might be distribution specific.